### PR TITLE
Prettier titlebars for dialogue states

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

File: go/base/static/css/conversations.less [hunk 1]
```diff
@@ -41,11 +41,16 @@
       .user-select(none);
 
       .titlebar {
+        font-size: 16px;
         padding: 8px 6px;
+
+        // Set the margin to the reverse of the state's padding so the titlebar
+        // actually looks like a titlebar
+        margin: -6px;
```
Comment (hodgestar) [inline go/base/static/css/conversations.less]: Maybe we can use a variable for this `6px` value?

Comment (hodgestar) [inline go/base/static/css/conversations.less]: Although we're not using variables anywhere else really and I don't think here is the right place to start.

**Comment by hodgestar** [comment]
:+1:

File: go/base/static/css/conversations.less (hunk 1 above)
Comment (justinvdm) [inline go/base/static/css/conversations.less]: Hmm, it does sound like a good idea though. I keep forgetting this is less :)

**Pull request merged**